openfl/utilities/optimizers/numpy/yogi_optimizer.py

```diff
@@ -0,0 +1,65 @@
+"""Adam optimizer module."""
+
+from typing import Dict
+from typing import Optional
+from typing import Tuple
+
+import numpy as np
+
+from .adam_optimizer import NumpyAdam
+
+
+class NumpyYogi(NumpyAdam):
+    """Yogi optimizer implementation.
+
+    Original paper:
+    https://papers.nips.cc/paper/8186-adaptive-methods-for-nonconvex-optimization
+    """
+
+    def __init__(
+        self,
+        params: Optional[Dict[str, np.ndarray]] = None,
+        *,
```

Comment:
Why are you using `*` not after `self`?

Reply:
@dmitryagapov, because `params` variable can be passed by position, not only by keyword, it's for highlighting  that 
the 'params' has more importance and if both the `params` and the `model_interface` are passed, then the initialization will be through the `params`.